README.md

```diff
@@ -38,8 +46,9 @@ synthea_omop_etl:
   target: dev
 ```
 
- 6. Ensure your profile is setup correctly from the command line:
+ 6. `cd` into the project directory, and ensure your profile is setup correctly using dbt debug:
 ```bash
+$ cd synthea_omop_etl
 $ dbt debug
 ```
```

Comment:
Should include `dbt deps` to install any required dependencies

Reply:
good catch, thank you!!!!